src/app/layout/components/SidebarRoutes.tsx

```diff
@@ -17,53 +17,53 @@ import {
     {
       name: "Dashboard",
       route: Routes.Dashboard,
-      icon: <DashboardIcon className="w-5 h-5 mr-3" />,
+      icon: <DashboardIcon className="w-3.5 h-3.5 mr-3" />,
```

Comment:
This seems wrong. When you loop over these items, you can apply classes there because all of them have same classes.

Reply:
Fixed